lib/nexmo_jwt/jwt_builder.rb

```diff
@@ -44,24 +46,24 @@ class JWTBuilder
     def initialize(params = {})
       Nexmo::JWTBuilder.validate_parameters_not_conflicting(params)
 
-      @application_id = set_application_id(params.fetch(:application_id))
-      @private_key = set_private_key(params.fetch(:private_key))
+      @application_id = application_id!(params.fetch(:application_id))
```

Comment:
Please revert the method name change, would be happy to consider these changes in a separate PR.